.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,137 @@
+name: "\U0001F41E Report a problem with MDN Plus"
+description: Report a problem with MDN Plus.
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: country
+    attributes:
+      label: Country
+      description: From what country are you accessing MDN Plus?
+      options:
+        - United States
+        - Canada
+        - Other
```

Comment:
I think we can get rid of this - if we keep it it should definetely be optional and say 'MDN' not 'MDN Plus'